tests/unit/accounts/test_services.py

```diff
@@ -104,11 +104,44 @@ def test_check_password_updates(self, db_session):
         ]
         assert user.password == "new password"
 
+    def test_create_user(self, db_session):
+        user = UserFactory.build()
+        email = "foo@example.com"
+        service = services.DatabaseUserService(db_session)
+        new_user = service.create_user(username=user.username,
+                                       name=user.name,
+                                       password=user.password,
+                                       email=email)
+        user_from_db = service.get_user(new_user.id)
+        db_session.flush()
```

Comment:
Ok, I'm OK with this flush then since there seems to be a good reason for it. Thanks for the explanation!